packages/orbit-components/src/tmp-tailwind/InputField/index.tsx

```diff
@@ -0,0 +1,318 @@
+          filled={!!value}
+          required={required}
+          error={!!error}
+          help={!!help}
+          labelRef={labelRef}
+          iconRef={iconRef}
+          onMouseEnter={() => setTooltipShownHover(true)}
+          onMouseLeave={() => setTooltipShownHover(false)}
+        >
+          {label}
+        </FormLabel>
+      )}
+      <div
+        className={cx(
+          "orbit-input-field-input-container",
+          "flex flex-row items-center justify-between",
+          "relative box-border",
+          "h-form-box-normal text-form-element-normal",
+          disabled ? "cursor-not-allowed" : "cursor-text",
+          disabled || readOnly
+            ? "text-form-element-disabled-foreground"
+            : "text-form-element-filled-foreground",
+        )}
+        ref={label ? null : labelRef}
+      >
+        {inlineLabel && !tags && (error || help) ? (
+          <Prefix>
+            {help && !error && (
+              <div className="flex" ref={iconRef}>
+                <InformationCircle color="info" size="small" />
+              </div>
+            )}
+            {error && (
+              <div className="flex" ref={iconRef}>
+                <AlertCircle color="critical" size="small" />
+              </div>
+            )}
+          </Prefix>
+        ) : (
+          prefix && <Prefix>{prefix}</Prefix>
+        )}
+        {label && inlineLabel && (
+          <div
+            className={cx(
+              "orbit-input-field-inline-label",
+              "flex items-center justify-center",
+              "pointer-events-none h-full",
+              "[&>span]:mb-0",
+              "[&>span]:text-form-box-normal [&>span]:whitespace-nowrap [&>span]:leading-normal",
+              "[&>span]:z-[3]",
+              !tags && (error || help) ? "ps-xxs" : "ps-sm",
+            )}
+            ref={labelRef}
+          >
+            <FormLabel
+              filled={!!value}
+              required={required}
+              error={!!error}
+              help={!!help}
+              inlineLabel={inlineLabel}
+            >
+              {label}
+            </FormLabel>
+          </div>
+        )}
+        {tags && <InputTags>{tags}</InputTags>}
+        {/* the rule is working weird, it passes only if the value is number, eg not even prop including number } */}
+        {/* eslint-disable-next-line jsx-a11y/aria-activedescendant-has-tabindex */}
+        <input
+          className={cx(
+            "font-base p-form-element-normal-padding text-inherit",
+            "z-[2] appearance-none border-none shadow-none",
+            "box-border h-full w-full min-w-0",
+            "cursor-auto",
+            "bg-transparent",
+            "flex-[1_1_20%]",
+            "[&::-webkit-inner-spin-button,&::-webkit-outer-spin-button]:appearance-none",
+            "[&::-webkit-inner-spin-button,&::-webkit-outer-spin-button]:margin-0",
+            "[&[data-com-onepassword-filled]]:bg-inherit",
```

Comment:
These selectors do not work in Tailwind. The margin prefix is `m-`.

```suggestion
            "[&::-webkit-inner-spin-button]:appearance-none [&::-webkit-outer-spin-button]:appearance-none",
            "[&::-webkit-inner-spin-button]:m-0 [&::-webkit-outer-spin-button]:m-0",
```

Reply:
Wrong line suggestion. Should be on the lines above